projects/js-packages/charts/src/components/bar-chart/bar-chart.tsx

```diff
@@ -70,8 +77,46 @@ const BarChartInternal: FC< BarChartProps > = ( {
 
 	const dataSorted = useChartDataTransform( data );
 
-	const chartOptions = useBarChartOptions( dataSorted, horizontal, options );
-	const defaultMargin = useChartMargin( height, chartOptions, dataSorted, theme, horizontal );
+	// Transform data to add a small value for zero bars to make them visible
+	// We'll store the original value for tooltips and calculations
+	const dataWithVisibleZeros = useMemo( () => {
```

Comment:
Could we please extract and export the functionality maybe as a hook so that it wouldn't bloat the bart chart because I don't see this one as a particularly common requirement 🤔

Reply:
Good call! I moved this to a shared hook in 7a4d4ce58419550bd0770df8e2c7266113a44c76